cn/docs/tutorials/kubernetes-basics/update-intro.html

```diff
@@ -26,9 +26,9 @@
 			<div class="col-md-8">
 			<h3>更新应用程序</h3>
 
-			<p>用户期望应用程序始终可用，并且开发人员有望每天部署新版本。这就是 Kubernetes 的滚动更新。 <b>Rolling updates</b> 允许通过使用新的 Pods 实例逐个更新来实现零停机的更新部署。新的 Pods 会被调度到可用资源的 Node 节点上。</p>
+			<p>用户期望应用程序始终可用，并且希望开发人员每天部署新版本。在 Kubernetes 上这通过滚动更新达成。 <b>Rolling updates</b> 允许通过使用新的 Pods 实例逐个更新来实现零停机的部署更新。新的 Pods 会被调度到可用资源的 Node 节点上。</p>
```

Comment:
similar comments， for specific technical names, like rolling update/rollout, I think at least we need to keep the English words